src/QMCWaveFunctions/Fermion/MatrixDelayedUpdateCUDA.h

```diff
@@ -376,7 +376,12 @@ class MatrixDelayedUpdateCUDA
   {
     // make this class unit tests friendly without the need of setup resources.
     if (!cuda_handles_)
+    {
```

Comment:
Ok I see this is a testing convenience behavior.  Rather see this in a test helper of some sort. Figuring out how to do that easily is a separate task.